cmd/lassie/fetch.go

```diff
@@ -132,7 +140,24 @@ func Fetch(cctx *cli.Context) error {
 
 	if len(fetchProviderAddrInfos) > 0 {
 		finderOpt := lassie.WithFinder(retriever.NewDirectCandidateFinder(host, fetchProviderAddrInfos))
+		if cctx.IsSet("ipniEndpoint") {
+			log.Warn("Ignoring ipniEndpoint flag since direct provider is specified")
+		}
 		lassieOpts = append(lassieOpts, finderOpt)
+	} else if cctx.IsSet("ipniEndpoint") {
```

Comment:
Can we add some unit tests for this new cmd flag?

Reply:
we currently don't have tests that go down in to the cli, unfortunately, we mostly do integration tests through the daemon so most of the code in here isn't touched

Reply:
We can use a similar setup as go-car [script test](https://github.com/ipld/go-car/blob/master/cmd/car/script_test.go) for lassie CLL